style.css

```diff
@@ -0,0 +1,5 @@
+h3 {
```

Comment:
- [ ] [REQUIRED] can you please use a class-selector instead of an element-selector. As required in the project requirements.